app/views/layouts/admin.html.slim

```diff
@@ -11,4 +11,7 @@ html
     = javascript_pack_tag 'application', 'data-turbolinks-track': 'reload'
   body.admin
     .container.admin-container
+      - flash_messages.each do |message_type, message|
```

Comment:
move this to partial. This could be used not only for admin side, but on user side as well